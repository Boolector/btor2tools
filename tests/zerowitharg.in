2 zero 1 1
