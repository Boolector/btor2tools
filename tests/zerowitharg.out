*** catbtor: parse error in 'zerowitharg.in' line 1: unexpected space (after no argument constant)
